bouachain/assetlist.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "bouachain",
+  "assets": [
+    {
+      "description": "The native token of the Bouachain blockchain, used for transaction fees, staking, and governance.",
+      "denom_units": [
+        {
+          "denom": "bouacoin",
+          "exponent": 8
+        },
+        {
+          "denom": "bouacoin",
+          "exponent": 8
+        }
+      ],
+      "base": "bouacoin",
+      "name": "BOUACOIN",
+      "display": "BOUA",
+      "symbol": "BOUA",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bouachain/images/bouachain.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bouachain/images/bouachain.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bouachain/images/bouachain.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bouachain/images/bouachain.svg",
+          "theme": {
+            "primary_color_hex": "#FF00C1"
+          }
+        }
+      ],
+      "socials": {
+        "website": "https://Bouachain.com",
+        "twitter": "https://x.com/bouachain"
+		"Github": "https://github.com/bouachain"
```

Comment:
If going to add github and telegram, please lowercase the G and the T. Also, the twitter line is missing the trailing comma